File: python/easy/find_the_difference.py
class Solution(object):
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        if len(t) <= len(s):
            return ret
        ds = collections.Counter(s)
        dt = collections.Counter(t)
        return (dt-ds).keys().pop()
        
        

File: cpp/easy/find_the_difference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        if(t.size() <= s.size())
            return NULL;
        char ret = 0;
        for(char m : s) ret ^= m;
        for(char n : t) ret ^= n;
        return ret;
    }
};


Conclustion:
easy, no any other comments
